examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-site/scripts/search.js

```diff
@@ -61,27 +61,50 @@
             }
         });
 
-        if (noResults) {
-            noResults.style.display = (total === count) ? "" : "none";
+        updateSearchResultsStatusMessageElement(token, foundComponentTeasers);
+    }
+
+    // useful for Accessibility, helping users with low vision and users with cognitive disabilities to identify the change in results
+    function updateSearchResultsStatusMessageElement(token, foundComponentTeasers) {
```

Comment:
Isn't it the same logic as for the Search v2? Can't it be reused here?

Reply:
It's not the same logic, just the names of the methods.